content/doc/addons/cellar.md

```diff
@@ -366,6 +376,55 @@ s3cmd delpolicy s3://<bucket-name>
 
 The original ACL should apply to all of your objects after that.
 
+### User access
+
+Cellar doesn't natively support creating different user accesses for the same add-on. Granting access to your Cellar add-on grants full access to all of your buckets. To grant limited access to a bucket, do the following:
```

Comment:
Maybe we can say this is currently a workaround and will be improved at some point?